client/landing/stepper/declarative-flow/internals/steps-repository/use-my-domain/index.tsx

```diff
@@ -114,12 +123,52 @@ const UseMyDomain: Step< {
 		}
 	};
 
+	const shouldHideButtons = isStartWritingFlow( flow );
+
+	if ( shouldUseStepContainerV2( flow ) ) {
+		const [ columnWidth, headingText, subHeadingText ] =
+			useMyDomainMode === 'domain-input'
+				? [
+						4 as const,
+						__( 'Use a domain I own' ),
+						__( 'Enter the domain you would like to use.' ),
```

Comment:
I asked a question about this sub title here p1742799582663609-slack-C08HKNPL2BG

But it's not a blocker.